### PR TITLE
cleanup old java code

### PR DESCRIPTION
**Pull request by addejans**
removed old comments and java code

### CODE REVIEW

File: app/src/main/java/com/example/textreccy/MainActivity.kt [hunk 1]
```diff
@@ -50,11 +46,7 @@ class MainActivity : AppCompatActivity() {
         super.onActivityResult(requestCode, resultCode, data)
         if (requestCode == REQUEST_CAMERA_CAPTURE && resultCode == RESULT_OK){
 
-//            lateinit var mCaptureRequestBuilder : Builder
```
Comment (Jeeze) [inline app/src/main/java/com/example/textreccy/MainActivity.kt:53]: We need to get the focus of the camera working.

**Review by Jeeze — APPROVED** [review]

**Review by Jeeze — CHANGES_REQUESTED** [review]
Testing denial of review.

**Comment by addejans** [comment]
> 
> 
> You code stinks

Why?

File: app/src/main/java/com/example/textreccy/MainActivity.kt (hunk 1 above)
Comment (addejans) [reply app/src/main/java/com/example/textreccy/MainActivity.kt:53]: I agree, we'll do this on a separate pull request.

Comment (Jeeze) [reply app/src/main/java/com/example/textreccy/MainActivity.kt:53]: No

**Review by Jeeze — CHANGES_REQUESTED** [review]

**Review by Jeeze — APPROVED** [review]

**Pull request merged**